### PR TITLE
Section html css branch

### PR DESCRIPTION
**Pull request by MICHAELKITH**
#What 
In this milestone we:
1.  Added index.html file
2. Added style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse ! </h1>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ]  As stated the project requirement Kindly make your added A .css file that includes `one class` with styles for your header (make it in your favorite color).

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @MICHAELKITH  , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linters error
✔️ You used correct GitHub Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ]  Good job for documenting your Readme File Professional, But I recommend to include "Getting Started" section and the section "Future Features", including 1-3 potential features to be added using the the [README template](https://github.com/microverseinc/readme-template)  

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELKITH** [comment]
the css file is added please check

**Comment by Abenezer-Tilahun** [comment]
Hello @MICHAELKITH  yeah .css file added but it must be with class with styles for your header, Just like `h1 class="header" Hello Microverse ! /h1`

**Comment by MICHAELKITH** [comment]
okay , i have changed bro you can check now

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MICHAELKITH, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

- [x] **As the last code reviewer suggested, please complete all the sections on the  README file to have a professional repo** 👇🏽 

> Good job for documenting your Readme File Professional, But I recommend to include "Getting Started" section and the section "Future Features", including 1-3 potential features to be added using the the [README template](https://github.com/microverseinc/readme-template)

## Optional suggestions

- [ ] **[OPTIONAL]** Please 🙏🏽 add a title and a description od the activities in your PR for a professional repo.

| Expected PR | Your PR |
|---|---|
| image | image |



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELKITH** [comment]
okay, should upload link for reviewal or you can review from here.

**Comment by MICHAELKITH** [comment]
@Abenezer-Tilahun  check if i have i done it

**Comment by Abenezer-Tilahun** [comment]
Hello @MICHAELKITH  yeah you done it, submit for review on your dashboard and it will be approved.

**Review by Qoosim — APPROVED** [review]
#   *Status: Approved* :tada: :heavy_check_mark: 

Hi @MICHAELKITH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Descriptive PR summary. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**